ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Node/TPraos.hs

```diff
@@ -297,6 +297,7 @@ protocolInfoTPraosShelleyBased ProtocolParamsShelleyBased {
           protVer
           genesis
           (shelleyBlockIssuerVKey <$> credentialss)
+          mempty -- TODO propagate?
```

Comment:
What's the hypothesis for why we wouldn't need to propagate?

Reply:
This function isn't ever called by `protocolInfoCardano` (which is the only place actually interesting to us), so I just did the lazy thing and didn't propagate it further for this prototype PR.

Reply:
I would consider addressing this if necessary for testing.